feedparser/http.py

```diff
@@ -145,7 +145,7 @@ def get(url, etag=None, modified=None, agent=None, referrer=None, handlers=None,
             if url_pieces.port:
                 new_pieces[1] = f'{url_pieces.hostname}:{url_pieces.port}'
             url = urllib.parse.urlunparse(new_pieces)
-            auth = base64.standard_b64encode(f'{url_pieces.username}:{url_pieces.password}').strip()
+            auth = base64.standard_b64encode(f'{url_pieces.username}:{url_pieces.password}'.encode('utf8')).strip()
```

Comment:
This can be directly encoded/decoded. No need to pass bytes around.
Also `str.encode()` and `bytes.encode()` use `utf-8` as encoding/decoding by default.
Additionally, this stripping at the end is redundant, because we get base64 encoded string as a result ...
```python
auth = base64.standard_b64encode(f'{url_pieces.username}:{url_pieces.password}'.encode()).decode()
```

Reply:
Fixed. All other encode/decode calls pass 'utf8', should they be changed in another PR?

Reply:
Yep, it will be better in a separate one, because it will be general refactoring unrelated to this specific fix.